backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -710,52 +683,45 @@ public UserInfo getUserByLoginEmail(String loginEmail) {
         throw new RestrictedAccessUserException();
       }
 
-      return consolidateUser(
-          apiUser,
-          oktaUser.getOrganizationRoleClaims(),
-          oktaUser.getStatus(),
-          oktaUser.isSiteAdmin());
+      return consolidateUser(apiUser, oktaUser);
     } catch (IllegalGraphqlArgumentException | UnidentifiedUserException e) {
       throw new OktaAccountUserException();
     }
   }
 
-  private UserInfo consolidateUser(
-      ApiUser apiUser,
-      Optional<OrganizationRoleClaims> optClaims,
-      UserStatus userStatus,
-      boolean isSiteAdmin) {
-
-    OrganizationRoleClaims claims = optClaims.orElseThrow(UnidentifiedUserException::new);
-
-    // use the target user's org so response is built correctly even if site admin is the requester
-    Organization org = _orgService.getOrganization(claims.getOrganizationExternalId());
```

Comment:
⚠️ Let me know if I am missing something but I found the logic from what was previously `lines 732-743` unnecessary because we can get `OrganizationRoles` from the `OrganizationRoleClaims` using [this `OrganizationService` method](https://github.com/CDCgov/prime-simplereport/blob/680ba6e8bf529f3d8d6ef6b277fd1deadb57472f/backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationService.java#L125).